From f568abfa47afb59ea20bed9c2de23b20330141c7 Mon Sep 17 00:00:00 2001
From: Fathy Boundjadj <hey@fathy.fr>
Date: Mon, 13 Feb 2023 17:18:18 +0100
Subject: [PATCH 1/2] Disable text rendering

---
 src/core/SkBitmapDevice.cpp | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/src/core/SkBitmapDevice.cpp b/src/core/SkBitmapDevice.cpp
index bf96c90371..b498d16d39 100644
--- a/src/core/SkBitmapDevice.cpp
+++ b/src/core/SkBitmapDevice.cpp
@@ -42,6 +42,8 @@
 
 #include <utility>
 
+#include "carbonyl/src/browser/bridge.h"
+
 class SkVertices;
 
 struct Bounder {
@@ -542,8 +544,10 @@ void SkBitmapDevice::drawImageRect(const SkImage* image, const SkRect* src, cons
 void SkBitmapDevice::onDrawGlyphRunList(SkCanvas* canvas,
                                         const sktext::GlyphRunList& glyphRunList,
                                         const SkPaint& paint) {
-    SkASSERT(!glyphRunList.hasRSXForm());
-    LOOP_TILER( drawGlyphRunList(canvas, &fGlyphPainter, glyphRunList, paint), nullptr )
+    if (carbonyl::Bridge::BitmapMode()) {
+        SkASSERT(!glyphRunList.hasRSXForm());
+        LOOP_TILER( drawGlyphRunList(canvas, &fGlyphPainter, glyphRunList, paint), nullptr )
+    }
 }
 
 void SkBitmapDevice::drawVertices(const SkVertices* vertices,
