From f4677bcc49983fa8aba7cef8746f78ff34a88cec Mon Sep 17 00:00:00 2001
From: Fathy Boundjadj <hey@fathy.fr>
Date: Mon, 13 Feb 2023 16:37:40 +0100
Subject: [PATCH] Disable GIO on Linux

---
 modules/portal/BUILD.gn | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/portal/BUILD.gn b/modules/portal/BUILD.gn
index de8a81be55..f9c647c30e 100644
--- a/modules/portal/BUILD.gn
+++ b/modules/portal/BUILD.gn
@@ -86,7 +86,7 @@ if ((is_linux || is_chromeos) && rtc_use_pipewire) {
     # `rtc_use_pipewire` is not set, which causes pipewire_config to not be
     # included in targets. More details in: webrtc:13898
     if (is_linux && !is_castos) {
-      defines += [ "WEBRTC_USE_GIO" ]
+      $ defines += [ "WEBRTC_USE_GIO" ]
     }
   }
 
